### PR TITLE
Add a client method to delete a single experiment

### PR DESCRIPTION
**Pull request by tomas-milata**

### CODE REVIEW

File: faculty/clients/experiment.py [hunk 1]
```diff
@@ -241,6 +241,24 @@ def list(self, project_id):
         endpoint = "/project/{}/experiment".format(project_id)
         return self._get(endpoint, ExperimentSchema(many=True))
 
+    def delete(self, project_id, experiment_id):
+        """Delete a specified experiment.
+
+        Parameters
+        ----------
+
```
Comment (acroz) [inline faculty/clients/experiment.py]: Would remove this line.

File: faculty/clients/experiment.py [hunk 2]
```diff
@@ -241,6 +241,24 @@ def list(self, project_id):
...
+        project_id : uuid.UUID
+        experiment_id : int
+
+        Returns
```
Comment (acroz) [inline faculty/clients/experiment.py]: Typically, if a method doesn't return anything (== returns None), we'd just omit the 'Returns' block.

File: tests/clients/test_experiment.py [hunk 3]
```diff
@@ -510,3 +510,15 @@ def test_log_run_data_empty(mocker):
...
     ExperimentClient._patch_raw.assert_not_called()
+
+
+def test_delete(mocker):
+    mocker.patch.object(ExperimentClient, "_delete_raw")
+
+    client = ExperimentClient(mocker.Mock())
+    result = client.delete(PROJECT_ID, EXPERIMENT_ID)
+
+    assert result is None
```
Comment (acroz) [inline tests/clients/test_experiment.py]: Similar to my comment on documentation, we'd not typically assert that the return value of a method in this case.

File: faculty/clients/experiment.py (hunk 2 above)
Comment (tomas-milata) [reply faculty/clients/experiment.py]: okay

**Pull request merged**